varinfo/cf_config.py

```diff
@@ -57,8 +57,7 @@ def __init__(
         self.mission = mission
         self.short_name = collection_short_name
 
-        self._cf_overrides: dict = {}
-        self._cf_supplements: dict = {}
+        self.cf_overrides: dict[str, dict[str, Any]] = {}
```

Comment:
Note - this is now a public class attribute. See the comment on `CFConfig.get_cf_overrides` for more information.